content/docs/studio/get-started.md

```diff
@@ -5,16 +5,10 @@ Studio].
 
 <admon>
 
-Iterative Studio [works very closely with your Git ecosystem][how-it-works]. So,
-you need to connect Iterative Studio to the Git repositories for your projects.
+We have renamed DVC Studio to Iterative Studio.
 
 </admon>
 
-> Note that we have renamed DVC Studio to Iterative Studio and Views to
-> Projects.
-
-https://www.youtube.com/watch?v=hKf4twg832g
-
```

Comment:
Why is the video removed? And the message about renaming is meant to accompany the video, which is why it's styled as a short note, not admon.

Reply:
Updated.